packages/core/src/Stores/client-store.js

```diff
@@ -2535,9 +2524,9 @@ export default class ClientStore extends BaseStore {
         }
     }
 
-    responseDerivezAvailableAccounts(response) {
+    responseCTraderAvailableAccounts(response) {
```

Comment:
We are changing the name here and this derivEz function used only in line 1710-1711.
So i think we can remove this whole function and its initialization.

Reply:
@hamza-deriv you're right, fixed.